standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java

```diff
@@ -30,4 +31,8 @@
    * Dumps platform state into logs
    */
   void dumpPlatformState();
+  /**
+   * Provides the raw config file from server startup
+   */
+  InputStream getPlatformConfiguration();
```

Comment:
Had not thought of the charset problem. Let's go with that, it is an easy thing to evolve should we decide to.